### PR TITLE
[RLlib] Enable connectors by default

### PR DESCRIPTION
**Pull request by ArturNiederfahrenhorst**
Signed-off-by: Artur Niederfahrenhorst <artur@anyscale.com>

## Why are these changes needed?

This PR tests what changes have to be made to default to connectors.

## Checks

- [x] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [x] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by gjoliver** [comment]
thanks for the PR. we can look at the failures together.

File: doc/source/ray-overview/installation.rst [hunk 1]
```diff
@@ -198,7 +198,7 @@ Ray has experimental support for machines running Apple Silicon (such as M1 macs
 
    * ``bash Miniforge3-MacOSX-arm64.sh``
 
-   * ``rm Miniforge3-MacOSX-arm64.sh # Cleanup.``
+   * ``rm https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-arm64.sh # Cleanup.``
```
Comment (wuisawesome) [inline doc/source/ray-overview/installation.rst:201]: this doesn't look very good...

File: doc/source/cluster/vms/user-guides/launching-clusters/aws.md [hunk 2]
```diff
@@ -94,21 +94,6 @@ setup_commands:
         sudo chmod 777 efs;
 ```
 
-### Configuring IAM Role and EC2 Instance Profile
```
[the remark this answers is not in the document]
Comment (ArturNiederfahrenhorst) [reply doc/source/cluster/vms/user-guides/launching-clusters/aws.md]: @wuisawesome Thanks for noticing. I merged https://github.com/ray-project/ray/pull/29205 which had the defect. Sorry for the trouble.

**Comment by ArturNiederfahrenhorst** [comment]
Since we are [enabling connectors by default before introducing compute_actions_from_raw_inputs](https://github.com/ray-project/ray/pull/30041), this PR is not needed anymore.

**Pull request closed without merging**